### PR TITLE
Fully style the app

### PR DESCRIPTION
**Pull request by codepantha**
Changes introduced via this pull request: 👇 
- Apply styling to the app navigation, book cards, and form.

### CODE REVIEW

File: src/components/Book/Book.js [hunk 1]
```diff
@@ -2,6 +2,7 @@ import React, { useEffect, useState } from 'react';
 import './Book.css';
 import { useDispatch, useSelector } from 'react-redux';
 import { uuid } from 'uuidv4';
+import { FaCircleNotch } from 'react-icons/fa';
 import { addBook, getBooks, removeBook } from '../../redux/books/books';
 
 const Book = () => {
```
Comment (D4nielJ) [inline src/components/Book/Book.js:8]: - [ ] It happens that when I go to the other tab and come back to books, the books are fetched again and append again to the list. This is unintended behavior and the styling should not affect it. Let's fix this bug so the books are stored in redux correctly and the action doesn't get dispatched again when the component re-renders.

Tip: I think we could do some check before fetching the books again.

bookstore-rerender

**Review by D4nielJ — CHANGES_REQUESTED** [review]
## Hello @codepantherr, 

You're doing great! 👏 
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights :sparkles: 
- Good UI! :rocket: 
- Books are being stored correctly on the API!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

## Happy coding and stay awesome! 🚀 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear or you can always find me on slack!

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/Book/Book.js (hunk 1 above)
Comment (codepantha) [reply src/components/Book/Book.js:8]: @D4nielJ Thank you 🙏 for your observation. The bug has been fixed.

**Review by dasileker — APPROVED** [review]
## **Hi @codepantherr,** 

## **Congratulations! 🎉 Project Approved ✅**
Your project is complete!  There is nothing else to say other than... it's time to merge it 🚀 
 


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by codepantha** [comment]
Thank you @dasileker

**Pull request merged**